src/evaluate/evaluate.ts

```diff
@@ -65,6 +65,10 @@ export class Evaluator {
       });
       this.context.addReference(new SimpleReference(outputId));
     });
+
+    this.context.template.transform.forEach((t) => {
+      this.context.stack.addTransform(t);
```

Comment:
What's the difference between adding it to stack directly and adding it to `ITemplateOption` [here](https://github.com/aws/aws-cdk/blob/v2.44.0/packages/@aws-cdk/core/lib/stack.ts#L1269)?

Reply:
Same thing, but I don't have to worry whether the array has been initialized:

https://github.com/aws/aws-cdk/blob/0358d51954b3ae32c6d7bdb490d498ab743770ec/packages/%40aws-cdk/core/lib/stack.ts#L726-L731